app/tests/util/webhooks.py

```diff
@@ -0,0 +1,55 @@
+import time
+import os
+from typing import List, Literal
+
+from httpx import AsyncClient
+import httpx
+
+from app.tests.util.string import get_wallet_id_from_JWT
+
+WEBHOOKS_URL = os.getenv("WEBHOOKS_URL", "http://localhost:3010")
+
+topics = Literal[
```

Comment:
This is redeclaring the topic from the webhook facade, maybe we can reuse that list to avoid inconsistencies in the future?